shopinvader_delivery_note/services/cart.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.addons.component.core import Component
+
+
+class CartService(Component):
+    _inherit = "shopinvader.cart.service"
+
+    def _validator_update(self):
+        """
+        Inherit to add the picking_note validator
+        :return: dict
+        """
+        validator = super(CartService, self)._validator_update()
+        validator.update({"picking_note": {"type": "string"}})
```

Comment:
@acsonefho the customer specify a 'delivery_note' the picpcking is for the preparation. This field should be named 'delivery_note'